frontend/src/Series/MonitoringOptions/MonitoringOptionsModalContent.css

```diff
@@ -0,0 +1,69 @@
+.container {
+  display: flex;
+}
+
+.year {
+  margin-left: 5px;
+  color: var(--disabledColor);
+}
+
+.poster {
+  flex: 0 0 170px;
+  margin-right: 20px;
+  height: 250px;
+}
+
+.info {
+  flex-grow: 1;
+}
+
+.overview {
+  margin-bottom: 30px;
+}
+
+.labelIcon {
```

Comment:
This is the only class being used, everything else can be nuked, otherwise looks good.

Reply:
done